ui-kit/src/main/kotlin/org/p2p/uikit/utils/skeleton/Shimmer.java

```diff
@@ -0,0 +1,344 @@
+/**
```

Comment:
Can you convert this file to Kotlin ?

Reply:
imho it will lead to more bugs. 
Let's leave it as java